validator/errorbundler.py

```diff
@@ -280,18 +283,25 @@ def _merge_messages(self, messages, callback, name):
             else:
                 trace.append(message["file"])
 
+            keys = ("column",
+                    "compatibility_type",
+                    "context",
+                    "description",
+                    "editors_only",
+                    "for_appversions",
+                    "line",
+                    "signing_severity",
+                    "tier")
+
+            kw = {key: message[key] for key in keys if key in message}
+
```

Comment:
Why write it like that? I don't find it more readable to be honest. Also, you added `editors_only` and `signing_severity`, is that on purpose?.